README.md

```diff
@@ -1,5 +1,9 @@
 # Comprehensive Rust 🦀
 
+[![Test](https://img.shields.io/github/actions/workflow/status/google/comprehensive-rust/build.yml?style=flat-square)](https://github.com/google/comprehensive-rust/actions/workflows/build.yml)
```

Comment:
You call this 'Test' here and 'Build workflow' in the other file. Maybe pick a consistent name?

Reply:
Yeah, definitely! I apparently changed my mind after copy-pasting the two blocks of code :smile: